src/Collector/EncountCollector.php

```diff
@@ -63,10 +63,16 @@ public function build($code, $description, $file, $line, $context)
         $this->url = $this->url();
         $this->ip = $this->ip();
         $this->referer = env('HTTP_REFERER');
-        $this->requestParams = Router::getRequest()->getAttributes();
         $this->session = isset($_SESSION) ? $_SESSION : array();
         $this->environment = $_SERVER;
         $this->cookie = $_COOKIE;
+
+        $request = Router::getRequest();
+        if (!is_null($request)) {
+            $this->requestParams = $request->getAttributes();
+        } else {
+            $this->requestParams = null;
```

Comment:
Thank you!
Please change the variable null to an empty array.

Reply:
Thanks for you reviewing, And I did it!